packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -45,6 +45,7 @@ function getZapierPayload(data: CalendarEvent & EventTypeInfo & { status?: strin
     title: data.title,
     description: data.description,
     customInputs: data.customInputs,
+    responses: data.responses,
     startTime: data.startTime,
```

Comment:
Looks good. But I would really like to have the same payload as our webhook for zapier hooks as well. What's the reason it's different cc @CarinaWolli

Reply:
I will change that: https://github.com/calcom/cal.com/issues/7797